cromwell

```diff
@@ -93,12 +95,12 @@ else
   if [ -n "$2" ]; then workflowID=$2; else workflowID=$(cat ${CROMWELL_LAST_WORKFLOW_FILE} ); fi
 fi
 echo "using workflow-id == $workflowID" 1>&2
-if [ "$1" = "status" ] ; then
+if [ "$1" = "status" ] || [ "$1" = "abort" ] ; then
    if [[ "$#" -le 2 ]] ; then
     status $workflowID
```

Comment:
won't this do the wrong thing for abort now?

Reply:
oops! right. now fixed